src/config/makeWebpackConfig.ts

```diff
@@ -41,30 +37,12 @@ export default function makeWebpackConfig({
 
   log('external packages %o', externalsRegex)
 
-  const builtInNode: NodeBuiltIn = {}
-  builtinModules.forEach(mod => {
-    builtInNode[mod] = 'empty'
-  })
-
-  builtInNode['setImmediate'] = false
-  builtInNode['console'] = false
-  builtInNode['process'] = false
-  builtInNode['Buffer'] = false
-
-  // Don't mark an import as built in if it is the name of the package itself
-  // eg. `events`
-  if (builtInNode[packageName]) {
```

Comment:
@pastelsky this has since changed in webpack. we now set `node: false` to tell it not to polyfill _any_ builtins. but losing this `if` concerns me, would you be able to explain what the intention of this whole block was (`builtInNode`) and if disabling it entirely will suffice?